code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -289,6 +289,7 @@ var/list/robot_verbs_default = list(
 			module_sprites["Basic"] = "robot_old"
 			module_sprites["Android"] = "droid"
 			module_sprites["Default"] = "robot"
+			module_sprites["Noble-STD"] = "Noble-STD"

```

Comment:
Hah, I can change it if you want me to, but that is the normal abbreviation of standard!